data/remnant/remnant outfits.txt

```diff
@@ -216,6 +216,19 @@ outfit "EMP Torpedo"
 	"emp torpedo capacity" -1
 	description "This torpedo can emit electromagnetic pulses, interfering with a ship's generators."
 
+outfit "EMP Torpedo Storage Bay"
+	category "Ammunition"
+	licenses
+		Remnant
+	cost 98000
+	thumbnail "outfit/emp rack"
+	"mass" 5
+	"outfit space" -12
+	"energy capacity" 400
+	"emp torpedo capacity" 7
+	description "The EMP Torpedo Storage Bay is used to store extra ammunition for EMP Bays."
```

Comment:
```suggestion
	description "The Storage Bay is used to store extra EMP Torpedoes."
```
Given that the torpedoes are used by the Ka'het as well, I think this fits better